### PR TITLE
BZ is done

### PR DESCRIPTION
**Pull request by BZhawaii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,17 @@ console.log(second_array) // [1, 2, 3]
...
+  let bucket1 =arr1;
+  console.log("bucket1 " + bucket1)
+  arr1 =  arr2;
+  console.log("arr1" + arr1)
+  arr2=bucket1;
+  console.log("arr2" + arr2)
+  first_arr = arr1;
+  second_array = arr2;
+  return(arr1, arr2);
+
```
Comment (BrooksPatton) [inline README.md:33]: This function does not need to return anything. It needs to use the fact that arrays are passed by reference to complete it.

**Review by BrooksPatton — CHANGES_REQUESTED** [review]
When you have the code working in a repl please add it here.

**Comment by BZhawaii** [comment]
I submitted the changes to this quick check. 

    On Thursday, January 12, 2017 7:47 PM, Brooks Patton <notifications@github.com> wrote:
 

 @BrooksPatton requested changes on this pull request.When you have the code working in a repl please add it here.In README.md:> @@ -20,7 +20,17 @@ console.log(second_array) // [1, 2, 3]
 
 ```js
 function swap(arr1, arr2) {
-  // -- YOUR ANSWER HERE --
+ 
+  let bucket1 =arr1;
+  console.log("bucket1 " + bucket1)
+  arr1 =  arr2;
+  console.log("arr1" + arr1)
+  arr2=bucket1;
+  console.log("arr2" + arr2)
+  first_arr = arr1;
+  second_array = arr2;
+  return(arr1, arr2);
+
This function does not need to return anything. It needs to use the fact that arrays are passed by reference to complete it.—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by BrooksPatton** [comment]
👍

**Review by BrooksPatton — APPROVED** [review]

**Pull request closed without merging**